NEWS.txt

```diff
@@ -251,6 +251,8 @@ Upgrading
       four digits need to be prefixed with a plus or minus sign.
     - cqlsh now returns a non-zero code in case of errors. This is a backward incompatible change so it may
       break existing scripts that rely on the current behavior. See CASSANDRA-15623 for more details.
+    - CASSANDRA-14902 updated the default compaction_throughput_mb_per_sec to to 64. The original
```

Comment:
nit: can we do the `See CASSANDRA-14902 for more details.` at end approach please.